index.html

```diff
@@ -37,6 +37,48 @@
     </nav>
 
 
+    <main>
+        <div class="card">
+            <div class="flex">
+                <div>
+                    <h1> Meow in! </h1>
+                    <p> You know that feeling when you miss sunshine?
+                        Yep, <em> that </em> is how much we missed you. <strong> Welcome back</strong>.
+                    </p>
+                </div>
+                <img src="assets/singing-barista.svg" alt="logo" width="250" height="100" />
+            </div>
+            <form>
+                <label>
+                    <span class="mb-07 block"> Your email </span>
+                    <input class="login-input" type="text" name="email" required>
+                </label>
+
+                <label>
+                    <span class="mb-07 mt-07 block"> Memorable password (yep, that one) </span>
+                    <div class="flex mb-07">
```

Comment:
The page is invalid. You cannot have `div` in the `label`.

Also, the label shouldn't contain the buttons. Let's refactor the code and connect the `label` with the `input` using the `for` attribute.